### DISCUSSION TITLE
Failed to resolve module specifier "@web3modal/ethers"

### DISCUSSION THREAD

**Discussion by Sesto93** [WalletConnect/walletconnect-monorepo#4278] [Developer Support]
Hello, I have followed all the instruction in the official documentation here: https://docs.walletconnect.com/web3modal/javascript/about 
I've done exactly the same thing, and I still gave this error "Uncaught TypeError: Failed to resolve module specifier "@web3modal/ethers". Relative references must start with either "/", "./", or "../"."
Please I really need help..

**Comment by glitch-txs** [comment]
Hi, we you need to use a bundler like vite to run the project with npm.
